3_javascript/javascript1/week2/homework javascript1 week2/event.js

```diff
@@ -0,0 +1,8 @@
+//Event application //
```

Comment:
It seems some functionality here is missing: `getEventWeekday` should print the weekday, e.g. "Thursday", that lies `days` into the future. The homework sheet has some hints, but feel free to reach out if it causes issues. 😄